USER_MANUAL.md

```diff
@@ -340,7 +352,163 @@ _TODO_
 
 ## 8\. Program Inputs
 
-Content for the "Program Inputs" section goes here.
+### &lt;parameter_level&gt; (programs)
+
+**Data Type:** String
+
+**Default input:** 'programs'
+
+**Description:** A string indicating the parameters in file are at the program settings level
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** Must be set to ```parameter_level: programs``` for an output setting parameter file.
+
+### &lt;version&gt; (programs)
+
+**Data type:** String
+
+**Default input:** 4.0
+
+**Description:** Specify version of LDAR-Sim. See section _[Versioning of Parameter Files](#versioning-of-parameter-files)_ for more information.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Improper versioning will prevent simulator from executing.
+
+### &lt;program_name&gt;
+
+**Data Type:** String
+
+**Default input:** "default"
+
+**Description:** The name of the program. Typical naming convention is to include "P_" before a name.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Each program must have a unique program name. If names are duplicated, they will override each other.
+
+### &lt;method_labels&gt;
+
+**Data Type:** List[strings]
+
+**Default input:** []
+
+**Description:** A list of the methods used within the program. For example, the following will use the aircraft and the OGI_FU methods:
```

Comment:
Can you provide a second example here using list brackets and explain that both ways of doing it are valid inputs